man/fi_domain.3.md

```diff
@@ -666,6 +667,78 @@ Applications can set the mr_cnt on input to fi_getinfo, in order to
 indicate their memory registration requirements.  Doing so may allow the
 provider to optimize any memory registration cache or lookup tables.
 
+## Traffic Class (tclass)
+
+The default traffic class associated with the domain and any endpoints
+created within the context of the domain.
+
+Applications can set the tclass in the domain, in order to indicate
+a request for a specific default traffic class. Otherwise, if the application sets
+the tclass field to FI_TC_TYPE_UNSPEC, the provider will set the
+default traffic class for the domain to be FI_TC_BEST_EFFORT.
+
+Traffic classes can be set according to two different types of classifications
+available to libfabric: FI_TC_TYPE_LABEL or FI_TC_TYPE_DSCP. If the traffic class
+*type* field is set to FI_TC_TYPE_DSCP, the value field of the traffic class must be set
+to a DSCP value. If the traffic class *type* field is set to FI_TC_TYPE_LABEL, the
+value field of the traffic class must be set to a libfabric or provider-specific
+defined label of a traffic class.
+
+Traffic classes can be any of the following defined classes of traffic,
+or any provider-specific definitions.
+
+** FI_TC_BEST_EFFORT **
+** FI_TC_LOW_LATENCY **
+** FI_TC_DEDICATED_ACCESS **
+** FI_TC_BULK_IO_DATA **
+** FI_TC_SCAVENGER **
+** FI_TC_MANAGEMENT **
+** FI_TC_BULK_IPC_DATA **
+
+The general behavior provided for each traffic class is provided below.
+
+*FI_TC_BEST_EFFORT*
+: This is the default, shared class, carrying the traffic for a number of
```

Comment:
I would state that this is the default in the absence of any other local or fabric configuration.